lib/release_notes_bot/projects.ex

```diff
@@ -54,18 +54,18 @@ defmodule ReleaseNotesBot.Projects do
     # This key is used to update the Channels table so that way a client_id
     # is always associated with a particular channel.
     [key] = Map.keys(input)
-    slack_channel = String.split(key, ":")
+    slack_channel = String.split(key, ":") |> List.last()
     client_id = String.to_integer(input[key]["selected_option"]["value"])
 
     Channels.update(
-      Channels.get(slack_id: List.last(slack_channel)),
+      Channels.get(slack_id: slack_channel),
```

Comment:
These types of changes really belong in https://github.com/mojotech/captains-log-release-agent/pull/22/commits/dd7b11ccc580b6fd0b59b451c1107b7c951d2396. This change does not relate to `notify channel upon configuration` as a unit of work